testing/citest/spinnaker_testing/spinnaker.py

```diff
@@ -341,6 +341,32 @@ def __determine_host_platform(cls, bindings):
 
     return host_platform
 
+  @classmethod
+  def __determine_native_base_url(cls, bindings, default_port):
+    """Helper function to determine a native host platform's base URL.
+
+    The returned base URL may be None if bindings['NATIVE_BASE_URL'] and
+    bindings['NATIVE_HOSTNAME'] are both missing.
+
+    Args:
+      bindings: [dict] List of bindings to configure the endpoint
+          NATIVE_BASE_URL: The base URL to use, if given.
```

Comment:
Mention that if base_url is not provided then the URL will be constructed using NATIVE_HOSTNAME and NATIVE_PORT and assuming http (no need to parameterize the protocol)